aeon/dj_pipeline/report.py

```diff
@@ -442,6 +448,68 @@ def delete_outdated_plot_entries():
         tbl.delete_outdated_entries()
 
 
+@schema
+class VisitSummaryPlot(dj.Computed):
+    definition = """
+    -> VisitSummary
+    ---
+    pellet_count_plotly:             longblob  # Dictionary storing the plotly object (from fig.to_plotly_json())
+    wheel_distance_travelled_plotly: longblob
+    total_distance_travelled_plotly: longblob  
+    foraging_bouts_plotly:           longblob  
+    """
+
+    key_source = dj.U("experiment_name", "subject", "visit_start", "visit_end") & (
+        VisitEnd
+        & f'experiment_name="{experiment_name}"'
```

Comment:
I don't think we need to apply a restriction on `experiment_name` here.
This is downstream from `VisitSummary` - meaning for each entry in VisitSummary, we should be able to make a daily summary plot for it.
Unless you're intended to restrict this to only multi-day visit? if so we can specify that in visit_duration

Reply:
Isn't this plotting for `'exp0.2-r0'`  only?